### PR TITLE
Properly say disabled/enabled on console to prevent confusion

### PR DESCRIPTION
**Pull request by Nivaturimika**

### CODE REVIEW

File: src/gui/gui_console.cpp [hunk 1]
```diff
@@ -1486,9 +1486,17 @@ void ui::console_edit::edit_box_enter(sys::state& state, std::string_view s) noe
...
 		break;
-	case command_info::type::instant_research:
+	case command_info::type::instant_research: {
+		auto has_us = false;
+		for(const auto n : state.cheat_data.instant_research_nations)
+			if(n == state.local_player_nation) {
+				has_us = true;
+				break;
+			}
+		log_to_console(state, parent, !has_us ? "\xA7GEnabled" : "\xA7RDisabled");
```
Comment (schombert) [inline src/gui/gui_console.cpp:1496]: shouldn't this come from the localization csv file?

Comment (Nivaturimika) [reply src/gui/gui_console.cpp:1496]: logging to console works a bit different and cant make use of localization csv

Comment (Nivaturimika) [reply src/gui/gui_console.cpp:1496]: atleast not w/o discarding colour info

Comment (schombert) [reply src/gui/gui_console.cpp:1496]: yes you can; you can grab the csv entry and convert it to a string and then pass the string to the log to console function

Comment (Nivaturimika) [reply src/gui/gui_console.cpp:1496]: the idea is to have COLOR

Comment (Nivaturimika) [reply src/gui/gui_console.cpp:1496]: Now that I think of it, it would be more efficient to make use of the checkmark/cross symbols we already have

Comment (schombert) [reply src/gui/gui_console.cpp:1496]: what is wrong with "\xA7G" + to_string(csv_key) ?

Comment (Nivaturimika) [reply src/gui/gui_console.cpp:1496]: > what is wrong with "\xA7G" + to_string(csv_key) ?

a checkmark will do then

**Pull request merged**